todo_list_server/with_orm/models.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional, Generic, TypeVar
+from pydantic.generics import GenericModel
+
+from sqlmodel import Field, SQLModel
+
+
+class TodoBase(SQLModel):
+    title: str
+    content: str
+
+
+class Todo(TodoBase, table=True):
+    id: Optional[int] = Field(default=None, primary_key=True)
+    done: Optional[bool] = Field(default=False)
```

Comment:
```suggestion
    id: int | None = Field(default=None, primary_key=True)
    done: bool | None = Field(default=False)
```

뭔가 `done: bool | None = False` 도 되지 않을까 싶긴 한데... 제가 sqlmodel을 잘 몰라서 이건 은비님이 한번 테스트해봐 주세요

Reply:
https://docs.python.org/ko/3/library/typing.html#typing.Optional

문서에 따르면 `Optional[X]`가 `X | None` 타입과 동일하다고 하네요!

Reply:
https://github.com/eunnbi/zero-to-hero/pull/3/commits/8cca1dd8286091209b485d639d37c5c5527113af
https://github.com/eunnbi/zero-to-hero/pull/3/commits/6dd96b60c6d5e39557f9e59e2f2cfaf9362a356f

Reply:
맞아요. 3.9였나 어떤 버전인지는 모르겠는데, `Optional[X]` 를 `X | None` 으로 쓸 수 있게 된 버전이 있었어요. `Optional` 을 import할 필요가 없어서 저를 비롯한 대부분의 파이썬 개발자들은 후자를 선호합니다.